### PR TITLE
CAT-1161 Add math operation for floor and ceiling

### PR DESCRIPTION
**Pull request by obusher**
* Added Floor and Ceil functions along with tests.
* Removed the German Translation

Reopening of #1213 from @KKcorps. (+ Rebased, squashed, rewritten commit message .... )

First [yellow](https://jenkins.catrob.at/job/Catroid-Multi-Job-Custom-Branch-RELOADED/2042/) test-job

### CODE REVIEW

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

File: catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorListFragment.java [hunk 1]
```diff
@@ -77,6 +77,7 @@
 			R.string.formula_editor_function_round, R.string.formula_editor_function_mod,
 			R.string.formula_editor_function_arcsin, R.string.formula_editor_function_arccos,
 			R.string.formula_editor_function_arctan, R.string.formula_editor_function_exp,
+			R.string.formula_editor_function_floor,R.string.formula_editor_function_ceil,
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorListFragment.java]: checkstyle complains about missing whitespace here

File: catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorListFragment.java [hunk 2]
```diff
@@ -89,6 +90,7 @@
 			R.string.formula_editor_function_round_parameter, R.string.formula_editor_function_mod_parameter,
 			R.string.formula_editor_function_arcsin_parameter, R.string.formula_editor_function_arccos_parameter,
 			R.string.formula_editor_function_arctan_parameter, R.string.formula_editor_function_exp_parameter,
+			R.string.formula_editor_function_floor_parameter,R.string.formula_editor_function_ceil_parameter,
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorListFragment.java]: and here

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

**Comment by avihric** [comment]
ok to test

**Comment by avihric** [comment]
ping @wslany FYI

**Comment by wslany** [comment]
Thx!

**Pull request merged**